src/classes/issues-processor.ts

```diff
@@ -911,6 +932,59 @@ export class IssuesProcessor {
     return this.options.removeStaleWhenUpdated;
   }
 
+  private async _removeLabelsWhenUpdatedFromStale(
+    issue: Issue,
+    removeLabels: Readonly<string>[]
+  ): Promise<void> {
+    if (!removeLabels.length) {
+      return;
+    }
+
+    const issueLogger: IssueLogger = new IssueLogger(issue);
+
+    issueLogger.info(
+      `Removing all the labels specified via the ${this._logger.createOptionLink(
+        Option.RemoveLabelsWhenUpdatedFromStale
+      )}`
+    );
+
+    for (const label of removeLabels.values()) {
+      await this._removeLabel(issue, label);
+    }
+  }
+
+  private async _addLabelsWhenUpdatedFromStale(
+    issue: Issue,
+    addLabels: Readonly<string>[]
```

Comment:
```suggestion
    addLabels: ReadonlyArray<string>
```

Reply:
Not sure if we can modify the parameter this is passing into, but changing it to a ReadonlyArray gets type complaints: `The type 'readonly string[]' is 'readonly' and cannot be assigned to the mutable type 'string[]'.'